docs/whats-new/release-candidate-1.md

```diff
@@ -0,0 +1,834 @@
+
+## Telemetry
+
+Within .NET Aspire, [Telemetry](../fundamentals/telemetry.md) remains a vital aspect of .NET Aspire. In this release we're introducing many new features to the Telemetry service.
+
+### Improve telemetry filtering
+
+Traces can now be filtered with attribute values. For example, if you only want to view traces for one endpoint in your app, the `http.route` attribute on HTTP requests can be filtered to a specified value.
+
+Telemetry filtering also supports autocomplete of existing values. The **Added filter** dialog provides a combo box for selecting from values that dashboard has received. This feature makes it much easier to filter to real data and helps avoid typos by entered a value yourself.
+
+> [!VIDEO 1e158316-4048-4f16-9afa-883c949f8063]
+
+### Combine telemetry from multiple resources
+
+When a resource has multiple replicas, you can now filter telemetry to view data from all instances at once. Select the parent resource, labeled `(application)`.
+
+:::image type="content" source="media/telemetry-resource-filter.png" alt-text="Filter by all instances of a resource":::
+
+### Browser telemetry support
+
+The dashboard now supports OpenTelemetry Protocol (OTLP) over HTTP and cross-origin resource sharing (CORS). These features unlock the ability to send OpenTelemetry from browser apps to the .NET Aspire dashboard.
+
+For example, a browser-based single page app (SPA) can configure the [JavaScript OTEL SDK](https://opentelemetry.io/docs/languages/js/getting-started/browser/) to send structured logs, traces and metrics created in the browser to the dashboard. Browser telemetry is displayed alongside server telemetry.
+
+:::image type="content" source="media/dashboard-browser-telemetry.png" alt-text="Trace detail page with browser telemetry":::
+
+For more information on configuring browser telemetry, see [Enable browser telemetry](../fundamentals/dashboard/enable-browser-telemetry.md) documentation.
+
+## App Host (Orchestration)
+
+The [.NET Aspire app host](../fundamentals/app-host-overview.md) is one of the most important features of .NET Aspire. In this release we're introducing many new features to the app host.
+
+### Waiting for dependencies
+
+You can now specify that a resource should wait for another resource before starting. This can help avoid connection errors during startup by only starting resources when their dependencies are ready.
+
+```csharp
+var builder = DistributedApplication.CreateBuilder(args);
+
+var queue = builder.AddRabbitMQ("rabbit");
+
+builder.AddProject<Projects.WebApplication1>("api")
+       .WithReference(queue)
+       .WaitFor(queue); // Don't start "api" until "queue" is ready...
+
+builder.Build().Run();
+```
+
+When the app host starts, it waits for the `rabbit` resource to be ready before starting the `api` resource.
```

Comment:
It is a little confusing to call the `queue/rabbit` resource with 2 names. Sometimes we call it "queue" like in the comment above, and sometimes we call it `rabbit` like on line 233.

Reply:
```suggestion
var rabbit = builder.AddRabbitMQ("rabbit");

builder.AddProject<Projects.WebApplication1>("api")
       .WithReference(rabbit)
       .WaitFor(rabbit); // Don't start "api" until "rabbit" is ready...

builder.Build().Run();
```

When the app host starts, it waits for the `rabbit` resource to be ready before starting the `api` resource.
```